### ISSUE TITLE
Accessibility: the available links "toggle"

### ISSUE THREAD

**Issue by afercia** [voldemortensen/menu-customizer#91]
Same as the search results: Posts, Pages, Categories, etc. present a list of available items and ideally should be marked up as lists. Currently, each item is a `dl` missing a required `dd` element.

Same considerations about the text "Add Menu Item" and the suggested technique.

**Comment by ocean90** [comment]
The `dl`/`dl` markup seems to come from `wp-admin/nav-menus.php`. We should fix it there too.

**Comment by ocean90** [comment]
Introduced in https://core.trac.wordpress.org/changeset/14248/trunk/wp-admin/includes/nav-menu.php, 5 years ago.

**Comment by designsimply** [comment]
Moved to https://core.trac.wordpress.org/ticket/32724
